src/providers/token-properties-provider.ts

```diff
@@ -64,14 +64,15 @@ export class TokenPropertiesProvider implements ITokenPropertiesProvider {
 
     const addressesToFetchFeesOnchain: string[] = [];
     const addressesRaw = this.buildAddressesRaw(tokens);
```

Comment:
Is `buildAddressesRaw` method no longer being used now? If so, I think we can delete the code block.

Reply:
`buildAddressesRaw` is still used to iterate raw addresses